fetch.bs

```diff
@@ -2728,26 +2730,59 @@ functionality.
+a <a for=/>list</a> of <a for="/" data-lt="fetch record">fetch records</a>.
 
-<p>A <a for="fetch group">fetch record</a> has an associated
-<dfn export for="fetch record" id=concept-fetch-record-request>request</dfn> (a
-<a for=/>request</a>).
+<p>A <a for=fetch>fetch group</a> has an associated
+<dfn export for="fetch group" id=concept-defer=fetch-record>deferred fetch records</dfn>,
+a <a for=/>list</a> of <a data-lt="deferred fetch record">deferred fetch records</a>.
 
-<p>A <a for="fetch group">fetch record</a> has an associated
-<dfn export for="fetch record" id=concept-fetch-record-fetch>controller</dfn> (a
-<a for=/>fetch controller</a> or null).
+<p>A <dfn export>fetch record</dfn> is a [=struct=]. It has the following items:
+
+<dl>
+ <dt><dfn export for="fetch record">request</dfn>
+ <dd>A <a for=/>request</a>.
+
+ <dt><dfn export for="fetch record">controller</dfn>
+ <dd>A <a for=/>fetch controller</a> or null.
+</dl>
+<hr>
+
+<p>A <dfn>deferred fetch record</dfn> is a <a for=/>struct</a> used to maintain state needed
+to invoke a fetch at a later time, e.g., when a {{Document}} object is unloaded or becomes
+not <a for=Document>fully active</a>. It has the following <a for=struct>items</a>:
+
+<dl>
+ <dt><dfn for="deferred fetch record">request</dfn>
+ <dd>A <a for=/>request</a>.
+
+ <dt><dfn for="deferred fetch record">done</dfn> (default false)
+ <dd>A boolean.
+</dl>
+
+<p>Each <a>navigable container</a> has an associated number
+<dfn>reserved deferred-fetch quota</dfn>. Its possible values are
+<dfn for="reserved deferred-fetch quota">minimal quota</dfn>, which is 8 kibibytes,
+<dfn for="reserved deferred-fetch quota">normal quota</dfn>, which is 64 kibibytes, or 0. Unless
+stated otherwise, it is 0.
 
 <hr>
 
-<p>When a <a for=fetch>fetch group</a> is
-<dfn export for="fetch group" id=concept-fetch-group-terminate>terminated</dfn>, for each associated
-<a for="fetch group">fetch record</a> whose <a for="fetch group">fetch record</a>'s
-<a for="fetch record">controller</a> is non-null, and whose <a for="fetch record">request</a>'s
-<a>done flag</a> is unset or <a for=request>keepalive</a> is false,
-<a for="fetch controller">terminate</a> the <a for="fetch group">fetch record</a>'s
-<a for="fetch record">controller</a>.
+<p>When a <a for=fetch>fetch group</a> <var>fetchGroup</var> is
+<dfn export for="fetch group" id=concept-fetch-group-terminate>terminated</dfn>:
+
+<ol>
+ <li><p><a for=list>For each</a> <a for="fetch group">fetch record</a> <var>record</var> of
+ <var>fetchGroup</var>'s <a for="fetch group">fetch records</a>, if <var>record</var>'s
+ <a for="fetch record">controller</a> is non-null and <var>record</var>'s
+ <a for="fetch record">request</a>'s <a>done flag</a> is unset and <a for=request>keepalive</a> is
+ false, <a for="fetch controller">terminate</a> <var>record</var>'s
+ <a for="fetch record">controller</a>.
+
+ <li><p><a>Process deferred fetches</a> for <var>fetchGroup</var>.
```

Comment:
> Process deferred fetches for fetchGroup.

Although this is when a document is terminating... There should be a step of "set activated to true" such that FetchLaterResult still gets the proper `activated` state after deferred fetch record being processed (previous invoke state approach has it right I think).

Reply:
Done